TUTORIAL.rst

```diff
@@ -0,0 +1,112 @@
+PDFmorph Tutorial
+#################
+
+Welcome! This will be a quick tutorial to accquaint users with PDFmorph
+and some of what it can do. 
+
+As we described in the README and installation instructions, please make
+sure that you are familiar with working with your command line terminal
+before using this application.
+
+Before you've started this tutorial, please ensure that you've installed
+all necessary software and dependencies.
+
+PDFmorph Example
+----------------
+
+	1. Open your Terminal or Command Prompt.
+
+	2. It it's not active already, activate your PDFmorph-equipped 
+	   conda environment by typing in ::
+	
+		source activate <pdfmorph_env>
+
+	   on Linux or ``activate <pdfmorph_env>`` on Windows.
+
+		* If you need to list your available conda environments,
+		  run the command ``conda info --envs`` or 
+		  ``conda env list``
+
+	3. Change directories with the ``cd`` command to a directory
+	   containing PDF files. For instance, I will use the file
+	   "ni_qmax25.cgr", found `here<https://github.com/diffpy/diffpy.pdfmorph/tree/master/tests/testdata>`_,
+	   so I will run ``cd ~/diffpy.pdfmorph/diffpy/pdfmorph/tests/testdata``
+
+	4. Using this file for an example (you can download it from the
```

Comment:
Rather than write it like this I would say before.

"In this tutorial we will use test data suppled as an example in the PDFmorph package

1. download the data by typing ....
1. move it to a directory where you want to work, e.g.,.....
1. cd to that directory....

and so on, so don't make it general, make it specific.

Then take a moment to describe what is in that folder.   .cgr files are calculated PDFs from PDFgui, .gr files are measured PDFs, then some words about what was measured, where it was measured, under what conditions and what temperatures etc.  Also acknowledge the experimenters if there is no paper reference.

Reply:
Can you tell me if this same issue happens for you? When I branched out form origin/master, I didn't see any available "tests" folder in `diffpy.pdfmorph/diffpy/pdfmorph`. That's the only reason I was hesitant of including this.

Reply:
Maybe it would be better to include some different, more clearly documented pdf data in a different folder called "tutorialdata" or something? I don't have any PDF data on hand and I'm not sure that it would be okay to use some off the internet, and I can't discern the information you're describing in the files we have in the repo already. Would it be possible for someone to merge a tutorialdata dir into the homepage of the repo with some .gr files and let me know the info so I can include that? Also, is there some better alternative to having them download the specific file without needing to clone the whole repository? Sorry for all the questions; I want to come up with a good solution!